CONTRIBUTING.md

```diff
@@ -49,3 +49,65 @@ We use [semantic-release](https://www.npmjs.com/package/semantic-release), which
 ## Feature Requests
 
 Make the case for a feature via an issue with a good title. The feature should be discussed and given a target inclusion milestone or closed.
+
+## Migrations
+
+When adding any fields to `models` in the `data-schema`, you will need to add a migration file. Sequelize-cli migrations keep track of changes to the database, helping with adding and/or reverting the changes to the DB. They also ensure models and migration files are in sync.
+
+### Migration files
+
+Create new migration files for any new DDL changes. Do not edit or remove migration files even after it's migrated and available in the database. 
+
+Reference links: 
+1. https://sequelize.org/master/manual/migrations.html
+2. https://github.com/sequelize/cli/tree/master
+
+_CAUTION: Create migrations user (sd_migrator) in DB and this user should be given restrictive privileges to perform DDL operations. Be careful when reverting migrations, as this may end up in table getting DELETED if the user has PRIVILEGES.
```

Comment:
```suggestion
_CAUTION: Create migrations user (sd_migrator) in DB and this user should be given restrictive privileges to perform DDL operations. Be careful when reverting migrations, as this may end up in table getting DELETED if the user has PRIVILEGES._
```